### PR TITLE
Error Feedback

### PR DESCRIPTION
**Pull request by kentwills**
Gives a better log for failures.

### CODE REVIEW

File: tests/spec/dirty_tests.py [hunk 1]
```diff
@@ -26,24 +27,32 @@ def clean_service(service_path):
...
+            for pid in lsof(service_path):
+                try:
+                    os.kill(pid, 10)
+                except OSError as error:
+                    if error.errno == 3:  # no such process
+                        pass
+                    else:
+                        raise
+                #with ignored(OSErorr(3, 'No such process')):
+                #    os.kill(pid, 10)
```
Comment (bukzor) [inline tests/spec/dirty_tests.py]: dead code

**Comment by bukzor** [comment]
failed circle

**Comment by bukzor** [comment]
i've merged this to my feature branch 
please no rebase/amend before this point 5e49270

**Comment by kentwills** [comment]
v1.1.0

**Pull request merged**